.github/actions/kind/action.yml

```diff
@@ -6,16 +6,23 @@ inputs:
     description: "Hostname of the main kind node"
     required: false
     default: kind-control-plane
+  node-image:
+    description: "Node image to be used"
+    required: false
+    default: kindest/node:v1.22.2
+  registry:
+    description: "Configures an insecure registry on kind-registry:5000 to be used with KinD"
+    type: boolean
+    default: false
 
 runs:
   using: "composite"
   steps:
     - name: Set up KinD
-      uses: container-tools/kind-action@v2
```

Comment:
I hit an issue using this action when running on runner using AWS linux:
```
Run container-tools/kind-action@v2
  with:
    config: .github/kind-config.yaml
    registry: true
    node_image: kindest/node:v1.22.2
  env:
    NAMESPACE: mynamespace
    KIND_MAIN_NODE_HOSTNAME: kind-control-plane
    JAVA_HOME: /actions-runner/_work/_tool/Java_Temurin-Hotspot_jdk/11.0.17-8/x64
    JAVA_HOME_11_X64: /actions-runner/_work/_tool/Java_Temurin-Hotspot_jdk/11.0.17-8/x64
/actions-runner/_work/_actions/container-tools/kind-action/v2/main.sh: line 66: args[@]: unbound variable
```
Haven't tried to fix or report this. It may be worth reporting and in case it is fixed to migrate back to this action. As a workaround we can use current approach of installing KinD by ourselves.